### PR TITLE
Add animations support in mobile app

### PR DESCRIPTION
**Pull request by jonathan-marmelab**
- [X] Rotate remaining tile animation

Peek 17-03-2023 10-28

### CODE REVIEW

File: mobile/src/board/BoardTypes.ts [hunk 1]
```diff
@@ -9,6 +9,11 @@ export interface BoardListItem {
   remainingSeats: number;
 }
 
+export interface UserAction {
+  kind: string;
+  payload: any;
```
Comment (JulienMattiussi) [inline mobile/src/board/BoardTypes.ts:14]: would be nice if your payload type could be more precise
something like RotationPayload | PlacementPayload | MovePayload

File: mobile/src/board/BoardHooks.ts [hunk 2]
```diff
@@ -19,18 +19,81 @@ export function useBoard(id: number | string): [Board | null, any | null] {
...
+                ...board.state,
+                gameState,
+              },
+            };
+          });
+
+          setTimeout(resolve, 0);
+        }),
+    ],
+    ["PLACE_TILE", () => fetchBoard()],
```
Comment (JulienMattiussi) [inline mobile/src/board/BoardHooks.ts:84]: I think you could simply write : 
`["PLACE_TILE", fetchBoard],`and avoid redefining an anonymous proxy function
same for the next line

File: mobile/src/board/BoardPages.tsx [hunk 3]
```diff
@@ -190,6 +190,7 @@ export function GetById() {
       <BoardView
         remainingTile={
```
Comment (JulienMattiussi) [inline mobile/src/board/BoardPages.tsx:191]: its a little confufing having one time a prop "remainingTile" that is a component, and elsewhere another variable named "remainingTile" whice is a boolean
Maybe you should change the name of the boolean for something like "hasRemainingTile"

**Review by JulienMattiussi — CHANGES_REQUESTED** [review]

File: mobile/src/board/BoardPages.tsx (hunk 3 above)
Comment (jonathan-marmelab) [reply mobile/src/board/BoardPages.tsx:191]: Will change in future PR

**Pull request merged**